src/main/java/uk/gov/pay/api/resources/telephone/TelephonePaymentNotificationResource.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.pay.api.resources.telephone;
+
+import com.codahale.metrics.annotation.Timed;
+import io.dropwizard.auth.Auth;
+import io.swagger.annotations.Api;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.api.auth.Account;
+import uk.gov.pay.api.model.telephone.CreateTelephonePaymentRequest;
+
+import javax.validation.Valid;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path("/")
+@Api(value = "/", description = "Public Api Endpoints to receive Telephone Payments")
+public class TelephonePaymentNotificationResource {
+
+    private static final Logger logger = LoggerFactory.getLogger(TelephonePaymentNotificationResource.class);
+    
+    public TelephonePaymentNotificationResource(){
+        
+    }
+    
```

Comment:
logger and empty constructor are unused and can be removed (but feel free to do this in follow up PR)